### PR TITLE
NetworkPools

### PR DESCRIPTION
**Pull request by paurosello**
Documentation for NetworkPools

### CODE REVIEW

File: src/content/basics/networkpools/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- Extending installation default CIDR range.
+- Limit the CIDR range a single cluster can have.
+
+## Lifecycle
+
+NetworkPools must be created before the cluster is created via Management Cluster Kubernetes API.
+
+NetworkPools can be modified as long as the CIDR doesn't overlap with other NetworkPools and contains the original CIDR. For example 10.0.0.0/16 could be modified to 10.0.0.0/15 but not to 10.0.0.0/17.
+
+## Assigning clusters to NetworkPools {#assigning-networkpool}
```
Comment (xh3b4sd) [inline src/content/basics/networkpools/index.md:34]: What is `{#assigning-networkpool}` for? Should that render in markdown?

**Review by xh3b4sd — APPROVED** [review]

File: src/content/basics/networkpools/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+description: A general description of network pools as a concept, it's benefits, and some details you should be aware of.
+date: 2021-01-09
+weight: 130
+type: page
+categories: ["basics"]
+user_questions:
+  - What is a network pool?
+  - What are network pools?
+  - In which cloud environments are network pools supported?
+  - Which releases introduced network pools?
```
Comment (njuettner) [inline src/content/basics/networkpools/index.md:12]: I think those two questions didn't have an answer.

File: src/content/basics/networkpools/index.md (hunk 1 above)
Comment (paurosello) [reply src/content/basics/networkpools/index.md:34]: I checked on nodepools and I think this is used to link directly to this paragraph, but we are not using it I will remove it

File: src/content/basics/networkpools/index.md (hunk 2 above)
Comment (paurosello) [reply src/content/basics/networkpools/index.md:12]: true, copy pasting from other entry but useful I will add this info

**Review by calvix — APPROVED** [review]

**Comment by marians** [comment]
Hi @paurosello ! I'd like to add our new platform support widget to the top of this page. (See https://docs.giantswarm.io/basics/nodepools/ for an example).

Can you tell me whether network pools are in alpha, beta, or GA as of AWS v12.7.0?

**Comment by paurosello** [comment]
They are GA, how do you create that widget?

**Comment by marians** [comment]
@paurosello Add this after the main headline:

```nohighlight
{{< platform_support_table aws="ga=v12.7.0" >}}
```

Documentation can be found in https://github.com/giantswarm/docs/blob/v1.19.13/CONTRIBUTING.md#list-of-shortcodes-with-explanation

**Pull request merged**